gherkin/ruby/.rsync

```diff
@@ -2,7 +2,3 @@
 ../../.templates/github/ .github/
 ../../.templates/ruby/.travis.yml .travis.yml
 ../testdata/ testdata/
-../download-gherkin-go scripts/download-gherkin-go
```

Comment:
Put this back in, so it gets updated if we make changes to `../download-gherkin-go`

Reply:
Done!

Reply:
Didn't look done, so I added this back on master.

Reply:
@aslakhellesoy Yeah, I had done it locally but hadn't pushed yet. Held off in favor of letting @enkessler have at it if he wanted, but for some reason my comment indicating that didn't make it? Womp. Thanks though!